typings/objection/index.d.ts

```diff
@@ -622,9 +651,10 @@ declare namespace Objection {
     isExecutable(): boolean;
     isFindQuery(): boolean;
 
-    runBefore(fn: (builder: this) => void): this;
+    runBefore(fn: (result: any, builder: this) => any): this;
```

Comment:
Is `any` the best we can do here for these?

Reply:
For runBefore the value can literally be anything. runAfter gets passed the query result, so we could improve that, but we would need to move the method to each of the different QueryBuilder classes. Return value can be anything.

Reply:
ok!

Reply:
I fixed the `runAfter` `result` type.